packages/core/src/components/product/ProductCard/ProductCard.tsx

```diff
@@ -55,7 +55,7 @@ export interface ProductCardProps {
 function ProductCard({
   product,
   index,
-  bordered = false,
+  bordered = true,
```

Comment:
Why are we changing this default?

Reply:
It has a more neutral design and don’t make us do fixes on the page layout to fits it. Example:

<img width="600" alt="image" src="https://github.com/vtex/faststore/assets/11613011/ecfb6fec-9036-4e68-88a8-b90959d19945">